website/blog/2024-05-14-monitor-ferretdb-performance-using-coroot.md

```diff
@@ -0,0 +1,180 @@
+---
+slug: monitor-ferretdb-performance-using-coroot
+title: 'Monitor FerretDB Performance using Coroot'
+authors: [alex]
+description: >
+  Observability is vital in any infrastructure. Find out how Coroot can provide real-time monitoring and visibility into a FerretDB setup.
+image: /img/blog/ferretdb-coroot.jpg
+tags: [tutorial, community, cloud, open source]
+---
+
+![Monitor FerretDB Performance using Coroot](/img/blog/ferretdb-coroot.jpg)
+
+Effective real-time monitoring is a critical aspect of any infrastructure.
+[Coroot](https://coroot.com/) is an open source observability platform that can provide real-time monitoring and visibility into a [FerretDB](https://www.ferretdb.com/) setup.
+
+<!--truncate-->
+
+Effective real-time monitoring is a critical aspect of any infrastructure.
+[Coroot](https://coroot.com/) is an open source observability platform that can provide real-time monitoring and visibility into a [FerretDB](https://www.ferretdb.com/) setup.
```

Comment:
Why do we repeat it twice?

![CleanShot 2024-05-14 at 21 07 02@2x](https://github.com/FerretDB/FerretDB/assets/11512/5d0b29f6-fba8-4ece-ac97-f802e7d5f5ea)

Reply:
This one wasn't solved